scipy/stats/_continuous_distns.py

```diff
@@ -7228,6 +7245,7 @@ class reciprocal_gen(rv_continuous):
 
     """
     def _argcheck(self, a, b):
+        raise_deprecation_warning()
```

Comment:
I think I tried that but i don't remember why I changed it. Why, though?

Reply:
> It would be good to raise a `DeprecationWarning` when `loguniform`/`reciprocal` are used, but it causes a lot of test failures. I seem to remember problems with filtering warnings in tests, so I thought I'd get some feedback before trying to do that

After thinking about this a bit, a deprecation might be too harsh. The parameterization of `log_uniform` is not very common and I am not sure if most people might want to use that. Instead, should we just raise a warning when the fit method is called? We can recommend users to use the `log_uniform` distribution and transform the parameters to `a`, `b` parametrization using `a = scale`, `b = c * a`, and `loc = loc`. Do you think that is enough?

Reply:
Maybe? Can you submit a PR to my branch so I can see?

Reply:
Done: mdhaber#81